### PR TITLE
change the term scratch cluster to draft cluster

### LINKED ISSUE #30448: Change the term `scratch cluster` to `draft cluster` or add some explanation/definition to it?
Issue with k8s.io/docs/tasks/access-application-cluster/configure-access-multiple-clusters/

The term `scratch cluster` that appears in the description might cause a bit of confusion (to me at least). Would it be better to rephrase it to something like `exercise cluster` or `draft cluster`?

CleanShot 2021-11-12 at 10 21 41@2x

### PR DESCRIPTION
**Pull request by taniaduggal**
Fix #30448

### CODE REVIEW

**Comment by RA489** [comment]
/retitle Change the term scratch cluster to draft cluster

**Comment by RA489** [comment]
/retitle change the term scratch cluster to draft cluster

**Comment by Sea-n** [comment]
There are many occurrence of `scratch` in this file.
```
44:Suppose you have two clusters, one for development work and one for SCRATCH work.
46:and your storage developers work in a namespace called `storage`. In your `SCRATCH` cluster,
49:to the SCRATCH cluster requires authentication by username and password.
63:  name: SCRATCH
75:  name: exp-SCRATCH
86:kubectl config --kubeconfig=config-demo set-cluster SCRATCH --server=https://5.6.7.8 --insecure-skip-tls-verify
107:kubectl config --kubeconfig=config-demo set-context exp-SCRATCH --cluster=SCRATCH --namespace=default --user=experimenter
129:  name: SCRATCH
142:    cluster: SCRATCH
145:  name: exp-SCRATCH
214:Now suppose you want to work for a while in the SCRATCH cluster.
216:Change the current context to `exp-SCRATCH`:
219:kubectl config --kubeconfig=config-demo use-context exp-SCRATCH
223:the `SCRATCH` cluster. And the command will use the credentials of the user
224:listed in the `exp-SCRATCH` context.
226:View configuration associated with the new current context, `exp-SCRATCH`.
332:    cluster: SCRATCH
335:  name: exp-SCRATCH
```

Could you explain a little bit about why just one of them needs to be changed?

**Comment by taniaduggal** [comment]
@Sea-n . All i have to changed it to draft?

**Comment by reylejano** [comment]
I've asked a clarifying question in the related issue https://github.com/kubernetes/website/issues/30448
Also, there is a second iteration of "scratch cluster" in the same paragraph that needs be modified.

**Comment by reylejano** [comment]
The commit message is invalid, it has a special character. You can modify the commit message and force push the updated message

**Comment by reylejano** [comment]
See this comment from the issue creator: https://github.com/kubernetes/website/issues/30448#issuecomment-1165154588

**Comment by kbhawkey** [comment]
Hello @taniaduggal .
This pull request has an invalid commit and reviewers have questions about the changes.
I will place a hold on this pull request, but feel free to cancel the hold when you are ready.
/hold
